src/localstack.js

```diff
@@ -10,11 +10,12 @@ const { buildConnectionAndConfig, waitForReady } = require('./utils/awsUtils');
 const { Writable } = require('stream');
 const logger = require('./utils/logging').getLogger('localstack');
 
-class TempWriteBuffer extends Writable {
+class LocalstackWriteBuffer extends Writable {
   constructor (resolve, container) {
     super();
-    this.reset();
```

Comment:
It looks like this may have been the last usage of `reset` - perhaps it could be deleted?

Reply:
If the buffer is exposed, it would make sense to allow a client to reset the buffer on demand.